packages/manager/modules/up-sell-cross-sell/rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import rollupConfig from '@ovh-ux/component-rollup-config';
+
+const config = rollupConfig({
+  input: 'src/index.js',
+});
+
+const outputs = [config.es()];
+
+if (process.env.BUILD === 'production') {
+  outputs.push(config.cjs());
+  outputs.push(
+    config.umd({
+      output: {
+        globals: {
+          angular: 'angular',
+        },
+      },
+    }),
+  );
```

Comment:
Package is private so I don't think we need to build all those dist , `esm` should be enough